src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/GH1001.ProvideRequiredLocationProperties_Invalid.sarif

```diff
@@ -5,7 +5,32 @@
     {
       "tool": {
         "driver": {
-          "name": "SARIF Functional Testing"
+          "name": "SARIF Functional Testing",
```

Comment:
>"name" [](http://example.com/codeflow?start=10&length=6)

All changes of GH1001-GH1007 in this folder,
you do NOT need to look details,
because I manually compared each of them against
the version before your change is merged into main,
and they are 100% the same.
So it is basically a revert,
which is what we wanted.